styles/google-classroom/catppuccin.user.css

```diff
@@ -0,0 +1,1145 @@
+        .UQuaGc {
+            color: @subtext0;
+        }
+
+        /*switch*/
+        .LXctle.VfPpkd-scr2fc-OWXEXe-gk6SMd:enabled .VfPpkd-uMhiad::after {
+            background: @accent-color!important;
+        }
+
+        .LXctle.VfPpkd-scr2fc-OWXEXe-gk6SMd:enabled:hover:not(.VfPpkd-ksKsZd-mWPk3d-OWXEXe-AHe6Kc-XpnDCe) .VfPpkd-Qsb3yd::before,
+        .LXctle.VfPpkd-scr2fc-OWXEXe-gk6SMd:enabled:hover:not(.VfPpkd-ksKsZd-mWPk3d-OWXEXe-AHe6Kc-XpnDCe) .VfPpkd-Qsb3yd::after,
+        .LXctle.VfPpkd-scr2fc-OWXEXe-gk6SMd:enabled:active .VfPpkd-Qsb3yd::before,
+        .LXctle.VfPpkd-scr2fc-OWXEXe-gk6SMd:enabled:active .VfPpkd-Qsb3yd::after {
+            background-color: @accent-color!important;
+        }
+
+        .LXctle.VfPpkd-scr2fc-OWXEXe-gk6SMd:enabled:hover:not(.VfPpkd-ksKsZd-mWPk3d-OWXEXe-AHe6Kc-XpnDCe):not(:active) .VfPpkd-uMhiad::after {
+            background: mix(black, @accent-color, 10%)!important;
+        }
+
+        .LXctle:enabled .VfPpkd-l6JLsf::after,
+        .LXctle:enabled:hover:not(.VfPpkd-ksKsZd-mWPk3d-OWXEXe-AHe6Kc-XpnDCe):not(:active) .VfPpkd-l6JLsf::after {
+            background: mix(white, @accent-color, 50%)!important;
+        }
+
+        .LXctle:enabled .VfPpkd-l6JLsf::before {
+            background: @overlay1!important;
+        }
+
+        .LXctle.VfPpkd-scr2fc-OWXEXe-uqeOfd:enabled .VfPpkd-uMhiad::after {
+            background: @text!important;
+        }
+
+        .LXctle.VfPpkd-scr2fc-OWXEXe-uqeOfd:enabled:hover:not(.VfPpkd-ksKsZd-mWPk3d-OWXEXe-AHe6Kc-XpnDCe):not(:active) .VfPpkd-uMhiad::after {
+            background: @overlay2!important;
+        }
+
+        .LXctle:enabled:hover:not(.VfPpkd-ksKsZd-mWPk3d-OWXEXe-AHe6Kc-XpnDCe):not(:active) .VfPpkd-l6JLsf::before {
+            background: @overlay0!important;
+        }
+
+        /* classroom title on hover */
+        .CbuVcc:focus .Pce5Kb,
+        .CbuVcc:hover .Pce5Kb {
+            color: @green;
+        }
+
+        .UvHKof .uO32ac,
+        .UvHKof .ypv4re,
+        .Iwp0Ue:not(:first-child),
+        .u73Apc {
+            border-color: @surface1;
+        }
+
+        /*loading*/
+        nav .bFjUmb-Wvd9Cc,
+        nav .bFjUmb-Wvd9Cc,
+        nav .CNpREd.bFjUmb-Wvd9Cc,
+        nav .CNpREd .bFjUmb-Wvd9Cc {
+            background-color: @accent-color;
+        }
+
+        nav .g2MItd.bFjUmb-Wvd9Cc,
+        nav .g2MItd .bFjUmb-Wvd9Cc,
+        nav .g2MItd.CNpREd.bFjUmb-Wvd9Cc,
+        nav .g2MItd.CNpREd .bFjUmb-Wvd9Cc {
+            background-color: @peach;
+        }
+
+        nav .UvHKof.bFjUmb-Wvd9Cc,
+        nav .UvHKof .bFjUmb-Wvd9Cc,
+        nav .UvHKof.CNpREd.bFjUmb-Wvd9Cc,
+        nav .UvHKof.CNpREd .bFjUmb-Wvd9Cc {
+            background-color: @surface1;
+        }
+
+        nav .g2MItd.bFjUmb-Wvd9Cc,
+        nav .g2MItd .bFjUmb-Wvd9Cc,
+        nav .g2MItd.CNpREd.bFjUmb-Wvd9Cc,
+        nav .g2MItd.CNpREd .bFjUmb-Wvd9Cc {
+            background-color: @peach;
```

Comment:
you could use some nesting here to avoid having to repeat yourself here. ie
```less
nav {
 .bFjUmb-Wvd9Cc {
 	color: @red;
 }
 .g2MItd.bFjUmb-Wvd9Cc {
 	color: @green;
 }
}
```